python/py_vapid/__init__.py

```diff
@@ -52,41 +43,48 @@ def __init__(self, private_key=None):
 
         """
         self.private_key = private_key
+        if private_key:
```

Comment:
Ah, I won't call `_private key`, but I will call `private_key`, this will raise a useful VapidException.